src/sentry/api/urls.py

```diff
@@ -2743,6 +2746,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         ProjectMonitorStatsEndpoint.as_view(),
         name="sentry-api-0-project-monitor-stats",
     ),
+    re_path(
+        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/autofix/codebase-index/status/$",
```

Comment:
we recently made a change in our api so that we support id and slug. we are slowly rolling out this changes and have recently rolled out support for projects. We should now make sure our apis reflect this change.
```suggestion
        r"^(?P<organization_slug>[^\/]+)/(?P<project_id_or_slug>[^\/]+)/autofix/codebase-index/status/$",
```